docs/guides/workflows_patterns/deployment_google_cloud_composer.rst

```diff
@@ -0,0 +1,155 @@
+========================================================================
+
+This guide will help you deploy Great Expectations within an Airflow pipeline running on Google Cloud Composer.
+
+.. admonition:: Prerequisites: This how-to guide assumes you have already:
+
+  - :ref:`Set up a working deployment of Great Expectations <tutorials__getting_started>`
+
+Steps
+-----
+
+Note: These steps are basically following the `Deploying Great Expectations with Airflow <https://docs.greatexpectations.io/en/latest/guides/workflows_patterns/deployment_airflow.html>`_ documentation with some items specific to Google Cloud Composer.
+
+
+1. Set up your composer environment
+
+    Create a composer environment using the `instructions located in the composer documentation <https://cloud.google.com/composer/docs/how-to/managing/creating>`_. Currently Airflow >=1.10.6 is supported by Great Expectations >=0.12.1.
+
+2. Create expectations
+
+    Create expectations using our guide to `Creating and Editing Expectations <https://docs.greatexpectations.io/en/latest/guides/how_to_guides/creating_and_editing_expectations.html>`_.
+
+    You can store your expectations anywhere that is accessible to the cloud composer environment. One simple pattern is to use a folder in the bucket provided by the composer environment. You can manually push updated expectation json files from your version controlled repository via ``gsutil`` or the GCS UI, or automate using Google Cloud Build or any other automation tool.
+
+    .. code-block:: bash
+
+        # copy expectation suites to bucket
+        # where COMPOSER_GCS_BUCKET is an environment variable with the name of your bucket
+        gsutil cp -r expectations/ gs://${COMPOSER_GCS_BUCKET}/great_expectations/
+
+3. Create your Data Context
+
+    Since we'd prefer not to use the airflow container filesystem to host a Data Context as a .yml file, another approach is to instantiate it in a python file either as part of your dag or imported by your dag at runtime. `Follow this guide on How to instantiate a Data Context without a yml file <https://docs.greatexpectations.io/en/latest/guides/how_to_guides/configuring_data_contexts/how_to_instantiate_a_data_context_without_a_yml_file.html>`_ and see the example below.
+
+    Note: You may want to reference our `Configuring metadata stores <https://docs.greatexpectations.io/en/latest/guides/how_to_guides/configuring_metadata_stores.html>`_ and `Configuring Data Docs <https://docs.greatexpectations.io/en/latest/guides/how_to_guides/configuring_data_docs.html>`_ how-to guides. All of the stores in the below example are configured to use GCS, however you can use whichever store is applicable to your infrastructure.
+
+    .. code-block:: python
+
+        project_config = DataContextConfig(
+            config_version=2,
+            plugins_directory=None,
+            config_variables_file_path=None,
+            datasources={
+                "my_pandas_datasource": {
```

Comment:
Might make sense here to explicitly add an example for a bigquery datasource with a credentials file URL - we should have that somewhere else as a separate article, but it might be a good start to include it here:

```            datasources={
                "sam_dev_bq": {
                    "credentials": {
                        "url": "bigquery://sam-dev/sam?credentials_path=/etc/airflow/env.d/credentials.json"
                    },
                    "class_name": "SqlAlchemyDatasource",
                    "module_name": "great_expectations.datasource",
                    "data_asset_type": {
                        "module_name": "great_expectations.dataset",
                        "class_name": "SqlAlchemyDataset"
                    }
                }
            },

Reply:
Thanks! Meant to hit that this morning. I see now that you already added, thank you :)